src/guide/basics/middleware.md

```diff
@@ -104,7 +104,7 @@ async def halt_response(request, response):
 ```
 :---
 
-## Order of execution
+#### Order of execution
```

Comment:
Why this and not H2?

Reply:
look at this file : [basics/headers.md](https://github.com/sanic-org/sanic-guide/blob/master/src/guide/basics/headers.md) `Tokens` and `Proxy headers` are part of `Headers`  and they are H4 title. 
In like manner, this also should be H4 title